src/plugins/workspace/public/components/workspace_creator/workspace_creator.tsx

```diff
@@ -145,7 +146,7 @@ export const WorkspaceCreator = (props: WorkspaceCreatorProps) => {
               ?.features[0].id;
             // Redirect page after one second, leave one second time to show create successful toast.
             window.setTimeout(() => {
-              if (isPermissionEnabled) {
+              if (isPermissionEnabled && goToCollaborators) {
```

Comment:
Should fix the `should redirect to workspace setting collaborators page if permission enabled` in `src/plugins/workspace/public/components/workspace_creator/workspace_creator.test.tsx` unit test failed due to this change.